apps/backend/samconfig.toml

```diff
@@ -0,0 +1,10 @@
+version = 0.1
+[default]
+[default.deploy]
+[default.deploy.parameters]
+stack_name = "sam-app"
```

Comment:
We don't need to commit this config file yet; if you really want to, I'd suggest making the `stack_name` more descriptive for our project like `constellation`